app/src/main/java/protect/card_locker/preferences/SettingsActivity.java

```diff
@@ -5,6 +5,7 @@
 import android.content.Intent;
 import android.os.Bundle;
 import android.view.MenuItem;
+import android.widget.Toast;
```

Comment:
This seems like a leftover from testing? I don't see Toast used anywhere

Reply:
Oops